bin/ui/messages/errors/noPackageJson.js

```diff
@@ -0,0 +1,12 @@
+import chalk from 'chalk';
+import dedent from 'ts-dedent';
+
+import { error } from '../../components/icons';
+
+export default () =>
+  dedent(chalk`
+    ${error} {bold No package.json found}
+    Chromatic only works from inside a JavaScript project.
+    Expecting a package.json somewhere up the directory tree.
```

Comment:
Optional: This is a sentence fragment in between two full sentences. May want to make this a full sentence as well.